internal/worker/modelworkermanager/modelworkermanager.go

```diff
@@ -240,22 +242,13 @@ func (m *modelWorkerManager) modelChanged(modelUUID string) error {
 	} else if err != nil {
 		return errors.Trace(err)
 	}
-	defer release()
-
-	if !isModelActive(model) {
```

Comment:
What's the situation here? Is the check done in the Model() call?

Reply:
Hi Ian, only activated model UUIDs would be passed here wrt the prev PR, one of the notes in the task is to remove this check.

https://warthogs.atlassian.net/browse/JUJU-7300